tests/system/fim/common_tasks/verify_alerts.yml

```diff
@@ -9,7 +9,7 @@
   - name: "Verify alert json log | {{ event }}"
     script: >
       verify_alerts_json.py -i {{ item.path }} -e {{ event }} 
-      -o {{ missing_alerts_json_path }}
+      -o {{ missing_alerts_json_path }} {{ verify_json_script_extra_params }}
```

Comment:
Let's standardize this as `json_verification_extra_arguments`

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/552/commits/9455e81de250cfb7a9077da377925afe70ae6782.